sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -170,7 +170,7 @@ export class AssessmentController {
         assessment,
         hasNOAApproval,
       );
-      const outputVariables = filterObjectProperties(
+      const outputVariables = await filterObjectProperties(
```

Comment:
There is an e2e test for assessment consolidated data. Not sure how it is passing. 
<img width="1061" height="761" alt="image" src="https://github.com/user-attachments/assets/07b19b6a-00d5-46a1-ba62-ffa4e9a7229e" />

Reply:
The "$" at the beginning is compatible with the new framework. I can remove from this to avoid confusion in the future.